### DISCUSSION TITLE
how to fully print my dict obj's members by `inspect()`?

### DISCUSSION THREAD

**Discussion by KarhouTam** [Textualize/rich#3549] [Q&A] [answered]
I want to use `inspect()` to print my parameter dict. But `inspect()` cannot print all members of it. 
 
![image](https://github.com/user-attachments/assets/a93c6515-07f7-4695-9f67-9fd0fe33fe13)

Can `inspect()` list it all? Not the abbreviation like ... +11.
 
What should I do?

**Comment by TomJGooding** [comment, marked as answer]
You might not need `inspect` if all you want to do is pretty print your dictionary?

https://rich.readthedocs.io/en/stable/pretty.html
